apis/r/R/SOMAExperiment.R

```diff
@@ -27,6 +32,30 @@ SOMAExperiment <- R6::R6Class(
         obs_query = obs_query,
         var_query = var_query
       )
+    },
+
+    #' @description Update the obs [`SOMADataFrame`] to add or remove columns.
+    #' See [`SOMADataFrame$update()`][`SOMADataFrame`] for details.
+    #' @param values A `data.frame`, [`arrow::Table`], or
+    #' [`arrow::RecordBatch`].
+    update_obs = function(values, row_index_name = NULL) {
+      self$obs$update(values, row_index_name)
+    },
+
+    #' @description Update the var `SOMADataFrame` to add or remove columns.
```

Comment:
Please add a comment saying the row count mustn't change & we'll fail it if they try

Reply:
Addressed in 99f611bc102c1337f2a3ae43ad1aee509f6b1bde